modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -1739,3 +1739,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
 	icon_state = "anubite_headpiece"
 	worn_icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
 	worn_y_offset = 4
+
+// Donation Reward for zcrow
+/obj/item/clothing/under/pants/custom_trackpants
+	name = "Tarin's custom sweatpants"
```

Comment:
oh yay i can help with this actually; names for items should be all lowercase, except for when in the loadout dm files for naming structure :)